athena-redis/src/test/java/com/amazonaws/athena/connectors/redis/integ/RedisIntegTest.java

```diff
@@ -346,21 +344,26 @@ private Stack getRedisStack()
      */
     private Endpoint getRedisInstanceData(String redisName, boolean isCluster)
     {
-        AmazonElastiCache elastiCacheClient = AmazonElastiCacheClientBuilder.defaultClient();
+        ElastiCacheClient elastiCacheClient = ElastiCacheClient.builder().build();
```

Comment:
ElastiCacheClient.create() does this in a slighly more elegant way

Reply:
@aimethed , changed.